sourcegraph/learn
# Style updates for featured content blocks on tag pages

### What should this PR do?
Resolves #441 by adjusting padding on header text for featured content blocks on tag pages.

Also takes another look at the `max-height` that got set in #417 and fixes linting (which was broken, due to renovate's auto updating of `eslint-plugin-unicorn`).

### Why are we making this change?
- We want styles to look good.

### What are the acceptance criteria? 
- Tag content blocks look correct in all browsers, old and new.

### How should this PR be tested?
- Check out the branch, and install deps with `npm ci`
- Check that linting works.
- Places to check: tag pages (`/tags/<whatever>`), homepage, index pages (`/posts`, `/videos`).
- Check out the deploy url in major browsers (Chrome, Firefox, Safari), as well as an older Safari, and check the above pages.


## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
